### ISSUE TITLE
Rate Limiting Option

### ISSUE THREAD

**Issue by guygastineau** [pa11y/pa11y-ci#210]
Is there a rate limiting option?  A vendor just implemented automatic rate limiting middleware for their servers, and an accessibility report we generate (using pa11y-ci to fetch all the errors) fails to get a lot of pages due to accessing pages too rapidly.  I was hoping to find a rate limiting option for the config, but I couldn't find it in the docs.

**Comment by guygastineau** [comment]
Hmm, I found the `wait` option in the main `pa11y` source.  I'm going to try that as a rate limiting hack.

**Comment by guygastineau** [comment]
Using the `wait` option in the config worked for mu use case.  It could be nice for other users if it were given as an example of rate limiting, but I think I can close this issue, as the title is not explicitly about documentation.
